Lib/test/test_ssl.py

```diff
@@ -960,6 +962,26 @@ def test_get_ciphers(self):
             len(intersection), 2, f"\ngot: {sorted(names)}\nexpected: {sorted(expected)}"
         )
 
+    def test_set_groups(self):
+        ctx = ssl.create_default_context()
+
+        # Test valid group list
+        self.assertIsNone(ctx.set_groups('P-256:X25519'))
+
+        # Test invalid group list
+        self.assertRaises(ssl.SSLError, ctx.set_groups, 'P-256:xxx')
```

Comment:
```suggestion
    def test_set_groups(self):
        ctx = ssl.create_default_context()
        self.assertIsNone(ctx.set_groups('P-256:X25519'))
        self.assertRaises(ssl.SSLError, ctx.set_groups, 'P-256:xxx')
```

We can be less verbose.

Reply:
Done.